File: angular/src/app/models/Event.ts

import {User} from "./User";
import {File} from "./File";

export class Event {
		closingTime: string;
		created: string;
		description: string;
		endTime: string;
		id: number;
		name: string;
		startTime: string;
		updated: string;

		attendees: User[];
		files: File[];
		speakers: User[];
}

File: mailer/src/main/java/ch/fhnw/edu/eaf/externalInterfaces/Event.java
package ch.fhnw.edu.eaf.externalInterfaces;

import java.util.Collection;
import java.util.Date;

/**
 * Created by apple on 19.05.17.
 */
public class Event {

    public Long id;
    public Date created;
    public Date updated;
    public String name;
    public String description;
    public String location;
    public Date startTime;
    public Date closingTime;
    public Date endTime;
    public Collection<User> speakers;
    public Collection<EventAttendee> attendees;

}
